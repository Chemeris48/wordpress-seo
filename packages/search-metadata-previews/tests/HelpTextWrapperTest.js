/* External dependencies */
import React from "react";
import renderer from "react-test-renderer";

/* Internal dependencies */
import HelpTextWrapper from "../src/snippet-preview/HelpTextWrapper";


describe( "HelpTextWrapper", () => {
	it( "matches the snapshot by default", () => {
		const component = renderer.create(
			<HelpTextWrapper helpTextButtonLabel="Open help" />
		);

		const tree = component.toJSON();
		expect( tree ).toMatchSnapshot();
	} );

	it( "matches the snapshot when the help text button is focused", () => {

	} );
} );
