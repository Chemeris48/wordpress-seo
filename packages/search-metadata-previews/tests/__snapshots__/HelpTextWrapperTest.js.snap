// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelpTextWrapper matches the snapshot by default 1`] = `
.c0 {
  width: 16px;
  height: 16px;
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
}

<div
  className="HelpTextWrapper__HelpTextContainer-sc-1lk9igo-0 FFpCR yoast-help"
>
  <button
    activeBackgroundColor="#f7f7f7"
    activeBorderColor="#000"
    activeColor="#000"
    aria-controls={null}
    aria-expanded={false}
    aria-label="Open help"
    backgroundColor="#f7f7f7"
    borderColor="#ccc"
    boxShadowColor="#ccc"
    className="Button__BaseButton-sc-32rbq-4,Button-sc-32rbq-3,Button-sc-32rbq-1,Button-sc-32rbq-2 Button-sc-32rbq-0 HelpTextWrapper__HelpTextButton-sc-1lk9igo-2 hWaXn khqyUK jSdTvi lmbUzU caGFwZ hbdXGw yoast-help__button"
    focusBackgroundColor="#fff"
    focusBorderColor="#0066cd"
    focusBoxShadowColor="#084a67"
    focusColor="#000"
    hoverBackgroundColor="#fff"
    hoverColor="#000"
    onClick={[Function]}
    textColor="#555"
    type="button"
  >
    <svg
      aria-hidden={true}
      className="c0 yoast-svg-icon yoast-svg-icon-question-circle HelpTextWrapper__StyledSvg-sc-1lk9igo-3 kJZwa-d"
      fill="#616161"
      focusable="false"
      role="img"
      size="16px"
      viewBox="0 0 1792 1792"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M1024 1376v-192q0-14-9-23t-23-9h-192q-14 0-23 9t-9 23v192q0 14 9 23t23 9h192q14 0 23-9t9-23zm256-672q0-88-55.5-163t-138.5-116-170-41q-243 0-371 213-15 24 8 42l132 100q7 6 19 6 16 0 25-12 53-68 86-92 34-24 86-24 48 0 85.5 26t37.5 59q0 38-20 61t-68 45q-63 28-115.5 86.5t-52.5 125.5v36q0 14 9 23t23 9h192q14 0 23-9t9-23q0-19 21.5-49.5t54.5-49.5q32-18 49-28.5t46-35 44.5-48 28-60.5 12.5-81zm384 192q0 209-103 385.5t-279.5 279.5-385.5 103-385.5-103-279.5-279.5-103-385.5 103-385.5 279.5-279.5 385.5-103 385.5 103 279.5 279.5 103 385.5z"
      />
    </svg>
  </button>
  <div
    className="animations__YoastSlideToggleContainer-cdldy9-0 gVphyr"
    duration={300}
  />
</div>
`;
